src/main.lua

```diff
@@ -1629,12 +1628,13 @@ end)
 addEventingHandler(ActionMap.SaveObservations, utils.hasMatchingTag("Action", ActionMap.SaveObservations), function(msg)
 	local reportTxId = msg.Tags["Report-Tx-Id"]
 	local failedGateways = utils.splitAndTrimString(msg.Tags["Failed-Gateways"], ",")
+	local epochIndex = msg.Tags["Epoch-Index"]
```

Comment:
Does the presence of this need to be asserted? Seems like it should because otherwise it will be used for a gte comparison in saveObservations (maybe should assert it there as well).

Reply:
added checks in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f

Reply:
Hmm this commit it not found.

Reply:
I see it in 11b9e1c3